sentinel2preprocessing/l1c_to_l2a_conversion.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import os
+import logging
+import time
+from typing import List
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+
+
+class Sentinel2Converter:
+    """
+    Class for converting Sentinel2 L1C to L2A images
+    """
+
+    def __init__(self, verbose):
+        """
+        :param verbose: bool, flag, print logging information, default: False
+        """
+        if verbose:
+            logger.setLevel(logging.INFO)
+        else:
+            logger.setLevel(logging.CRITICAL)
+
+    def convert_l1c_to_l2a(self, input_tile_path, output_dir_path) -> bool:
```

Comment:
Make func private, it is for internal usage

Reply:
Done:
https://github.com/QuantuMobileSoftware/sentinel2tools/pull/7/commits/e46872c0a9836c4039a3ea9fd6361c55751e7ed2